src/IsExternalInit.nuspec

```diff
@@ -47,16 +47,23 @@ Please see https://github.com/manuelroemer/IsExternalInit for additional informa
     <file src="ExcludeFromCodeCoverage/IsExternalInit.cs.pp" target="contentFiles/cs/net40/IsExternalInit/IsExternalInit.cs.pp"/>
     <file src="ExcludeFromCodeCoverage/IsExternalInit.cs.pp" target="content/net40/IsExternalInit/IsExternalInit.cs.pp"/>
 
-    <!-- >= .NET 5 supports IsExternalInit. -->
-    <file src="../../src/.nuget/_._" target="contentFiles/cs/net5.0"/>
-    <file src="../../src/.nuget/_._" target="content/net5.0"/>
+    <!--
+    >= .NET 5 requires IsExternalInit to be included per project.
+
+    See issue: https://github.com/manuelroemer/IsExternalInit/issues/5 for more details.
+    -->
+    <file src="ExcludeFromCodeCoverage/IsExternalInit.cs.pp" target="contentFiles/cs/net5.0/IsExternalInit/IsExternalInit.cs.pp"/>
```

Comment:
I think we can remove the `net5.0` folder(s) altogether since the `netstandard2.0` targets below will automatically include everything >= .NET 5 anyway. Or am I missing something here?